dev/breeze/src/airflow_breeze/commands/common_options.py

```diff
@@ -341,6 +341,15 @@ def _set_default_from_parent(ctx: click.core.Context, option: click.core.Option,
     "--use-uv/--no-use-uv",
     is_flag=True,
     default=True,
+    show_default=True,
```

Comment:
Not entirely familiar with this part, is this repeated on purpose this one and the next one, the only difference being the default?

Reply:
Yep. Ci Image has default "true", prod image "false" - CI image defaults can be changed but prod-image can be used by users and for now we should leave `pip` as default.